infra/modules/private-key-aws-parameter/variables.tf

```diff
@@ -12,3 +12,9 @@ variable "private_key" {
   type        = string
   description = "value of the key"
 }
+
+variable "user_managed_value" {
```

Comment:
I don't see where this is used. I assume is for discriminating use of `private-key-aws-parameter` vs `private-key-aws-parameter-user-managed` modules